Watchman.Discord/ReactionsService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+
+namespace Watchman.Discord
+{
+    public class ReactionsService
```

Comment:
napisz prosty przypadek użycia tego serwisu
żeby była jakaś funkcjonalność która to potwierdza 

np weryfikacja użytkowników
jeśli użytkownik określoną kliknie reakcje (wybierz jakąś, np kciuk w góre czy coś tego typu) w wiadomości `Zweryfikuj`
która będzie tylko jedna per serwer 
to dodajemy mu role "zweryfikowany"

aktualnie to może być dowolna rola, w przyszłości to będzie konfigurowalne

taka przykładowa funkcjonalność to dobry sposób na sprawdzenie czy koncept w ogóle ma sens w praktyce

Reply:
w sensie, że mam tak po prostu napisać jakiś kod, dzięki któremu mógłbym sprawdzić działanie tego mechanizmu? Jeśli tak to już wcześniej sprawdzałem i wszystko jest ok.

Reply:
potwierdzam 

chodzi właśnie o to czy sama obsługa w praktyce jest intuicyjna i nie jest problematyczna 
samo zrobienie `żeby działało` to nie jest jakiś problem

problemem jest zrobienie tego tak, żeby każdy czuł sie swobodnie używając tego 
i żeby nie okazało sie później w praktyce że połowe trzeba przebudować, bo nie przewidzieliśmy podstawowego scenariusza

Reply:
A z tą funkcjonalnością weryfikacji to w sumie może zostać - w przyszłości można będzie ją wykorzystać 

I będzie testem sprawdzającym czy mamy obsłużone wszystko czego potrzebujemy